addins/Cake.Prompt.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.Prompt
```

Comment:
There's been some discussion around this one in #609

Reply:
The discussion seems to be about someone taking over a project from the original author and whether this person asked for the original author's permission or not. This doesn't change the fact that there is a [package on NuGet](https://www.nuget.org/packages/Cake.Prompt/) which was automatically picked up by the AddinDiscoverer. 

What do you suggest we do? Blacklist this addin until the original author's permission is obtained maybe?

Reply:
@Jericho Permission has already been "granted" by virtue of the fact that the original repository was licensed using the MIT license.  What exactly would blacklisting do in this case?  Ultimately, PR 609 will likely be merged in.  If there are on going checks that need to happen for it, which would be skipped due to blacklisting, then I would suggest that we don't blacklist it.

Reply:
@gep13 I don't know. I was just trying to address @devlead comment. I'm not even sure what he is saying. If blacklisting is not the right solution, I'm open to suggestions.